tools/topology/m4/eq_fir_coef_flat.m4

```diff
@@ -1,5 +1,5 @@
 # Flat FIR EQ 19-Dec-2018
-CONTROLBYTES_PRIV(EQFIR_priv,
+CONTROLBYTES_PRIV(DEF_EQFIR_PRIV,
```

Comment:
Good, I'll change the default in filter blobs generator to this string so there will no more need to hand edit these.